app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -86,6 +86,11 @@
                     </a>
                 {% endif %}
             {% endif %}
+            {% if 'change_readerstudy' not in readerstudy_perms and 'read_readerstudy' in readerstudy_perms and object.is_educational and object.leaderboard_accessible_to_readers %}
+                <a class="nav-link" href="{% url 'reader-studies:leaderboard' slug=object.slug %}">
+                       <i class="fas fa-trophy fa-fw"></i>&nbsp;Leaderboard
+                    </a>
+            {% endif %}
```

Comment:
Editors also get the `read` permission, hence the double check on permissions here. I also considered moving the leaderboard tab down here for editors as well, but then it's put at the very bottom, and I think it's better to keep it where it is now for editors.

Reply:
I have opted to pull the leaderboard tab up to avoid checking for it in two places.